runtime/src/main/codestarts/quarkus/quakus-hilla-maven-codestart/base/pom.xml.tpl.qute

```diff
@@ -0,0 +1,94 @@
+<project>
+{#if hillaVersion }
+    <properties>
+        <hilla.version>{hillaVersion}</hilla.version>
+    </properties>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>dev.hilla</groupId>
+                <artifactId>hilla-maven-plugin</artifactId>
+                <version>$\{hilla.version\}</version>
+                <executions>
+                  <execution>
+                    <goals>
+                      <goal>prepare-frontend</goal>
+                    </goals>
+                  </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+
+    <profiles>
+        <profile>
+            <!-- Hilla Production mode is activated using -Pproduction -->
+            <id>production</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>dev.hilla</groupId>
+                        <artifactId>hilla-maven-plugin</artifactId>
+                        <version>$\{hilla.version\}</version>
+                        <executions>
+                            <execution>
+                                <goals>
+                                    <goal>build-frontend</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
```

Comment:
```suggestion
                </plugins>
                 <dependencies>
            <dependency>
                <groupId>com.vaadin</groupId>
                <artifactId>vaadin-core</artifactId>
                <exclusions>
                    <exclusion>
                        <groupId>com.vaadin</groupId>
                        <artifactId>vaadin-dev</artifactId>
                    </exclusion>
                </exclusions>
            </dependency>
        </dependencies>
```
(Wrongly formatted)
Maybe also exclude dev-server as mentioned in the docs: https://hilla.dev/docs/lit/guides/production/production-build

Reply:
Good point. I'll add the exclusion